src/gwt/src/org/rstudio/core/client/widget/FileOrUrlChooserTextBox.java

```diff
@@ -62,4 +96,53 @@ public String getText()
    {
       return getTextBox().getText();
    }
+   
+   @Override
+   public void onDetach()
+   {
+      checkTextBoxInterval_ = 0;
```

Comment:
If the timer was started, do we need to stop it on detach so it doesn't poll indefinitely? 

There's a global `Scheduler` that will make this a little easier (its `RepeatingCommand` callback lets you return false to end the polling).